.crowdin.yaml

```diff
@@ -7,14 +8,6 @@ files:
     source: '/modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-en_US.json'
     translation: '/modules/admin-ui-frontend/resources/public/org/opencastproject/adminui/languages/lang-%locale_with_underscore%.json'
 
-  -
-    source: '/modules/engage-theodul-core/src/main/resources/ui/language/en-US.json'
-    translation: '/modules/engage-theodul-core/src/main/resources/ui/language/%locale%.json'
-
-  -
-    source: '/modules/**/src/main/resources/static/language/en-US.json'
```

Comment:
I can see an argument for keeping this, although there are no *current* files matching this glob.  Should we keep it?

Reply:
If there isn't anything, let's remove it.